lib/danger/dangerfile_dsl.rb

```diff
@@ -58,18 +91,38 @@ def message(message)
       # When an undefined method is called, we check to see if it's something
       # that either the `scm` or the `request_source` can handle.
       # This opens us up to letting those object extend themselves naturally.
-      def method_missing(method_sym, *_arguments, &_block)
+      # This will also look for plugins
+      def method_missing(method_sym, *arguments, &_block)
+        # SCM Source
         if AvailableValues.scm.include?(method_sym)
-          # SCM Source
           return env.scm.send(method_sym)
         end
 
+        # Request Source
         if AvailableValues.request_source.include?(method_sym)
-          # Request Source
           return env.request_source.send(method_sym)
         end
 
-        raise "Unknown method '#{method_sym}', please check out the documentation for available variables".red
+        # Plugins
+        class_name = method_sym.to_s.danger_class
+        if Danger::Dangerfile::DSL.const_defined?(class_name)
+          plugin_ref = Danger::Dangerfile::DSL.const_get(class_name)
```

Comment:
Yeah, great point, I'll add this! Also, I should add a `.run` method to the super class that raises an exception if you didn't implement it